auth/auth.go

```diff
@@ -485,6 +496,8 @@ type Options2LO struct {
 	Audience string
 	// PrivateClaims allows specifying any custom claims for the JWT. Optional.
 	PrivateClaims map[string]interface{}
+	// UniverseDomain is the default service domain for a given Cloud universe.
+	UniverseDomain string
```

Comment:
Is this public field needed?

Reply:
Yes, because we are setting it in `auth/credentials/filetypes.go`